app/models/topic.rb

```diff
@@ -0,0 +1,14 @@
+# == Schema Information
+#
+# Table name: topics
+#
+#  id         :bigint           not null, primary key
+#  name       :string           not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class Topic < ApplicationRecord
+  validates :name, presence: true
+  has_one_base64_attached :avatar
+  scope :get_all, -> { limit(50) }
```

Comment:
maybe we can remove this scope as we don't need to limit the number of topics that we return

Reply:
agree, but we should paginate them

Reply:
Probably it is better to do the pagination in a separate PR, so you can just remove this scope now.